src/test/java/com/vackosar/gitflowincrementalbuild/mocks/RepoResolver.java

```diff
@@ -59,11 +63,4 @@ private Repository createRepo() {
         }
         return repo;
     }
-
-    @Override
-    public void close() throws Exception {
```

Comment:
`org.eclipse.jgit.transport.resolver.RepositoryResolver.open(C, String)` says:
> The caller is responsible for closing the returned Repository.
and in fact `org.eclipse.jgit.transport.DaemonService.execute(DaemonClient, String, Collection<String>)` *is* auto-closing each repo it has retrieved from `RepositoryResolver`.
So there is no point in closing the repositories explicitly.